Update wording and provide new screenshots

Tediore opened the pull request:
The onboarding page was out of date given the additions to the onboarding process. I provided some new screenshots and updated the description of the process.

I added some new screenshots to the automation page.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

Tediore committed:
Create trash

Tediore committed:
Add files via upload

Tediore committed:
Delete trash

Tediore committed:
Update wording and screenshots

Tediore committed:
Update screenshots and wording

Also improve clarity

Tediore commented:
Alright, I think I got it right this time 🙄 😆

klaasnicolaas commented:
Yes it went well 👍

Tediore commented:
I changed the default branch in my fork to `current`. Looks like the default is `next` when you fork it, at least on my end.

frenck commented on source/getting-started/automation.markdown:
```diff
@@ -6,36 +6,43 @@ redirect_from:
+Once your devices are set up, it's time to put the cherry on the pie: automation. In this guide we're going to create a simple automation rule to **turn on the lights when the sun sets**.
 
-In Home Assistant, open the menu by clicking on the top-left icon and click on configuration. Now click on automation. This is the automation screen from which you can manage all the automations in Home Assistant.
+In Lovelace, click Configuration in the sidebar, then click Automation. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```
This is not Lovelace specific.

```suggestion
In Home Assistant, click Configuration in the sidebar, then click Automation. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```

frenck commented on source/getting-started/onboarding.markdown:
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
+Set your username and password.
+</p>
+
+Next, you can enter a name for your home and set your location and unit system. Click DETECT to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
```
```suggestion
Next, you can enter a name for your home and set your location and unit system. Click "DETECT" to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
```

frenck commented on source/getting-started/onboarding.markdown:
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
+Discovery of devices on your network.
+</p>
+
+Finally, click Finish. Now you're brought to the main screen of Home Assistant called Lovelace. This screen will show all of your devices. So let's get that screen filled up!
```
The screen is not called Lovelace. Lovelace is only a part of the full frontend.

frenck commented on source/getting-started/onboarding.markdown:
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
 When each integration is done setting up, it will ask you to put the new devices in areas. Areas allow you to organize all the devices in your home.
 
-When you're done, navigate back to the states panel and voila, your devices are ready for you to control.
+When you're done, navigate back to Lovelace and voila, your devices are ready for you to control.
```
We should avoid Lovelace here. It is just the dashboard or Home Assistant frontend.
Lovelace is still optional + not the whole interface is Lovelace.

Tediore committed:
Revert improper references to Lovelace.

Tediore commented:
Thanks Frenck! Sorry for the misunderstanding. I reverted my changes on the last two comments--thoughts?

frenck commented:
Well, we currently have both the old state's screen and Lovelace for displaying information about your system. Besides that, there is the other things our frontend provides.

Maybe just call it the "Home Assistant web interface"? It covers it all right? To be clear on the subject, I'm not quite sure myself... I'm just opening up the discussion on this.

Tediore commented:
Right. I changed it to Lovelace since I was editing the onboarding page and Lovelace is the default. But yes, "Home Assistant web interface" would be a good general term!

I'll change that soon.

Tediore committed:
"states screen" > "Home Assistant web interface"

Tediore commented:
Alright, how about now? 😉

frenck commented:
Great progress! Now the thing left, is to fix the images... Please check the build preview 😉

Tediore commented:
It looks ok to me. [Am I looking in the wrong spot](https://github.com/home-assistant/home-assistant.io/blob/fdf733e2a813da258ab84ec8f790a46919a94b4b/source/getting-started/onboarding.markdown)?

frenck commented:
Yes, you are. The GitHub preview is an approximation, however, it is not accurate.
If you click on the Netlify build, you'll go to an actual build preview of the whole website, based on this PR.
You'll then notice the broken images.

Tediore commented:
Ah, thank you. I added those images to my fork and they're included in this PR. Are they not showing up because the PR hasn't been merged yet? If not, how should I resolve this?

Forgive my ignorance--still learning about Github.

frenck commented:
The paths in this repository do not match 1-1 with the generated result.

I would recommend:

- Move the images to `/source/images/getting-started/`
- Rename the images to have filenames that do not contain spaces or capital letters and are not numbered (order might change in the future) (e.g.,  `12 - new trigger.png` -> `new-trigger.png`)
- Use image refs: `/images/getting-started/new-trigger.png`

While you are in `/image/getting-started/`, that folder contains the "old" images, it would be nice if you could clean those up as well.

Tediore commented:
Will do! Sorry for all the trouble with this...

frenck commented:
Don't worry @Tediore! You are doing fine!

Tediore committed:
Merge pull request #1 from home-assistant/current

Current

Tediore committed:
Delete automation-new-action.png

Tediore committed:
Delete automation-new-blank.png

Tediore committed:
Delete automation-new-name-trigger.png

Tediore committed:
Delete integrations.png

Tediore committed:
Create trash

Tediore committed:
Add new screenshots

Tediore committed:
Delete 01 - username.png

Tediore committed:
Delete 02 - location.png

Tediore committed:
Delete 03 - location.png

Tediore committed:
Delete 04 - devices.png

Tediore committed:
Delete 05 - lovelace.png

Tediore committed:
Delete 06 - config.png

Tediore committed:
Delete 07 - integrations.png

Tediore committed:
Delete 08 - new integration.png

Tediore committed:
Delete 09 - config 2.png

Tediore committed:
Delete 10 - automation editor.png

Tediore committed:
Delete 11 - new automation.png

Tediore committed:
Delete 12 - new trigger.png

Tediore committed:
Delete 13 - action.png

Tediore committed:
Delete 14 - server control.png

Tediore committed:
Delete trash

Tediore committed:
Update screenshot paths

Tediore committed:
Update onboarding.markdown

Tediore commented:
Ok. Moved the new screenshots to the requested directory and removed the old screenshots. Just waiting on the netlify build to be ready to confirm the image paths are correct.

Tediore committed:
Correct image paths

Tediore committed:
Update automation.markdown

Tediore committed:
Reword and provide additional information

This information was outdated, especially with the addition of default_config. Hopefully these changes make it clearer for new users who are confused about the missing homeassistant: line in their default configuration (this is something that has been popping up in the forum lately).

I linked to an external source for the time zone names. Perhaps we should add this list to Home Assistant website instead (or if it already exists, my apologies: I can point it to that instead).

Tediore committed:
Update configuration.markdown

Tediore commented:
Also, I included the changes to `configuration.markdown` in this PR (from the other PR I closed).

frenck reviewed (approved):
Awesome job @Tediore! (And nicely recovered the PR as well!)  👍 

Thank you!
